expose-app-api/models/user.js

```diff
@@ -0,0 +1,26 @@
+// models/user.js
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../database.js';
+
+export const User = sequelize.define('User', {
+  username: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  created_at: {
+    type: DataTypes.DATE,
+    allowNull: false,
+    defaultValue: sequelize.literal('CURRENT_TIMESTAMP'),
+  },
+
+});
```

Comment:
let's add new lines to all files that have this symbol XD it's some antiquated github/git thing, I don't know why it is needed